web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -119,8 +125,18 @@ export const SidebarCardsList = (props: Props): ReactElement => {
 
   return (
     <>
-      <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
-      <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {isDesktopAndUp && (
+        <>
+          <h2 className="h2-styling margin-top-0 font-weight-normal">
+            {" "}
+            {Config.dashboardDefaults.sidebarHeading}
+            <span data-testid="for-you-counter" className="margin-left-05 text-base">
+              ({getForYouCardCount(business, props.certifications, props.fundings)})
+            </span>
+          </h2>
+          <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+        </>
+      )}
```

Comment:
[sand] It'd be nice to check this logic with a test. I don't think jest can actually adjust the screenwidth, but we could mock `isDesktopAndUp` to return `true/false`.